Source/SwiftLintFramework/Extensions/Array+SwiftLint.swift

```diff
@@ -67,4 +67,11 @@ extension Collection {
     var isNotEmpty: Bool {
         return !isEmpty
     }
+
+    /// Get the only element in the collection.
+    ///
+    /// If the collection is empty or contains more than one element the result will be `nil`.
+    var singleElement: Element? {
```

Comment:
Nit: I think `onlyElement` might be a better name but I don’t feel strongly.

Reply:
Agree, it's more on point.